packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -15,238 +15,236 @@ import PlaceholderComponent from '../Components/placeholder-component.jsx';
-                      component: (
-                          <Localize
-                              i18n_default_text='This trade type is currently not supported on {{website_name}}. Please go to <0>Binary.com</0> for details.'
-                              values={{
-                                  website_name,
-                              }}
-                              components={[
-                                  <a
-                                      key={0}
-                                      className='link link--orange'
-                                      rel='noopener noreferrer'
-                                      target='_blank'
-                                      href={urlFor('user/statementws', { legacy: true })}
-                                  />,
-                              ]}
-                          />
-                      ),
-                  };
-        } else if (row_obj.desc && ['deposit', 'withdrawal', 'adjustment'].includes(row_obj.action_type)) {
-            action = {
-                message: row_obj.desc,
-            };
-        }
+    return action;
+};
 
-        return action;
-    };
+const Statement = ({
+    account_statistics,
+    component_icon,
+    currency,
+    data,
+    date_from,
+    date_to,
+    error,
+    filtered_date_range,
+    handleDateChange,
+    handleScroll,
+    has_selected_date,
+    is_empty,
+    is_loading,
+    is_mx_mlt,
+    is_switching,
+    onMount,
+    onUnmount,
+}) => {
+    React.useEffect(() => {
+        onMount();
+        return () => {
+            onUnmount();
+        };
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
 
-    getAccountStatistics() {
-        const { account_statistics, currency } = this.props;
-        return (
-            <React.Fragment>
-                <div className='statement__account-statistics'>
-                    <div className='statement__account-statistics-item'>
-                        <div className='statement__account-statistics--is-rectangle'>
-                            <span className='statement__account-statistics-title'>
-                                {localize('Total deposits')} {`(${currency})`}
-                            </span>
-                            <span className='statement__account-statistics-amount'>
-                                <Money amount={account_statistics.total_deposits} currency={currency} />
-                            </span>
-                        </div>
+    const getAccountStatistics = () => (
+        <React.Fragment>
+            <div className='statement__account-statistics'>
+                <div className='statement__account-statistics-item'>
+                    <div className='statement__account-statistics--is-rectangle'>
+                        <span className='statement__account-statistics-title'>
+                            {localize('Total deposits')} {`(${currency})`}
+                        </span>
```

Comment:
is it possible to use Text instead of span for all spans here?

Reply:
It's already done in another PR.